### ISSUE TITLE
Right-click menu staying when mouse is moved away

### ISSUE THREAD

**Issue by eggbrow** [runelite/runelite#1623]
Haven't seen this done before in other clients. Might be quite useful for some users.

**This is a feature request.** Couldn't find where to add the appropriate label for it.

**Comment by Nightfirecat** [comment]
Labels are added by administrators, so you're fine as far as that goes.

That sounds like a pretty unique feature, but I wouldn't want that to be a default. How would this be managed? (modifier key, or some other control to toggle this behavior?)

**Comment by eggbrow** [comment]
Yeah definitely not active by default, as very few users would like it. It's something that anybody who would benefit from, would probably want to be able to toggle regularly. Probably an enable/disable in settings with config to add a toggle key like alt. Idk which keys are used for stuff already.
